zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -391,7 +394,9 @@ private void skipRecord(final LoggedEvent currentEvent) {
   private void readNextEvent() {
     if (shouldExport()) {
       final LoggedEvent currentEvent = logStreamReader.next();
-      if (eventFilter == null || eventFilter.applies(currentEvent)) {
+      if (eventFilter == null
+          || eventFilter.applies(currentEvent)
+              && !positionsToSkip.contains(currentEvent.getPosition())) {
```

Comment:
:wrench: We should reuse the `SkipPositionsFilter` I've built for skipping positions during processing.